.github/workflows/rust.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     - name: Fuse dependency
       run: sudo apt-get install -y libfuse-dev pkg-config
     - name: Download pak files
-      run: cd pak-examples & ./download.sh
+      run: cd pak-examples && ./download.sh
```

Comment:
The shell that is used by workflows is started with `-eo pipefail`, meaning you could also just use `;` or write:
```yaml
      run: |
        cd pak-examples
        ./download.sh
```